pilot/templates/__init__.py

```diff
@@ -88,7 +88,7 @@ def apply_project_template(
 
     last_development_step = project.checkpoints.get('last_development_step')
     if last_development_step:
-        project.save_files_snapshot(last_development_step['id'])
+        project.save_files_snapshot(last_development_step['id'], summaries=template.get("files"))
```

Comment:
No need to ask LLMs to describe files we already have in a template, just get summaries from the template metadata.